coral/coral/pkg/graphs/resource_models/License.json

```diff
@@ -674,26 +1001,6 @@
                     "visible": true,
                     "widget_id": "ff3c400a-76ec-11e7-a793-784f435179ea"
                 },
-                {
```

Comment:
This is important and shouldn't be removed

Reply:
its not actually removed its above. Ill mark where with another comment.